Makefile

```diff
@@ -1,3 +1,34 @@
 
-.DEFAULT:
-	fab "$@"
\ No newline at end of file
+down:
+	@docker-compose down
+
+build_image:
```

Comment:
Maybe we should use same syntax as in fabric to decrease potential future confusion? i.e. `buildnocache` for this one
Also, don't we need to execute `./ci_cd/version.sh` here?